concrete/attributes/user_selector/controller.php

```diff
@@ -0,0 +1,91 @@
+<?php
+namespace Concrete\Attribute\UserSelector;
+
+use Concrete\Core\Attribute\Controller as AttributeTypeController;
+use Concrete\Core\Attribute\FontAwesomeIconFormatter;
+use Concrete\Core\Entity\Attribute\Value\Value\NumberValue;
+use User;
+
+class Controller extends AttributeTypeController
+{
+    public function getIconFormatter()
+    {
+        return new FontAwesomeIconFormatter('user');
+    }
+
+    public function getAttributeValueClass()
+    {
+        return NumberValue::class;
+    }
+
+    public function form()
+    {
+        $value = null;
+        if (is_object($this->attributeValue)) {
+            $value = $this->getAttributeValue()->getValue();
+        }
+        if (!$value) {
+            if ($this->request->query->has($this->attributeKey->getAttributeKeyHandle())) {
+                $value = $this->createAttributeValue((int) $this->request->query->get($this->attributeKey->getAttributeKeyHandle()));
+            }
+        }
+        $this->set('value', $value);
+        $this->set('user_selector', $this->app->make('helper/form/user_selector'));
+    }
+
+    public function getDisplayValue()
+    {
+        $uID = $this->getAttributeValue()->getValue();
+        $user = User::getByUserID($uID);
+        if (is_object($user)) {
+            return t('<a href="%s">%s</a>', $user->getUserName(), $user->getUserPublicProfileUrl());
```

Comment:
The order of arguments to `t()` suggest it would result in something like `<a href="user name">/profile/user/1</a>`

Reply:
Why sprintf(), when a simple string concatenation will do?

Reply:
It's the same, I just wanted to suggest to not use `t` since there's nothing to be translated.